config/abilities/provider_any.rb

```diff
@@ -10,6 +10,8 @@
 
     can :manage, user
 
+    can(:manage, :policies) if account.tenant? && user.provider_can_use?(:policies)
```

Comment:
You only need to do it in 

* provider_admin.rb
`can(:manage, :policies)`

* provider_member.rb
`can(:manage, :policies) if user.has_permission(:policy_registry)`